packages/react-router/modules/__tests__/Router-test.js

```diff
@@ -2,6 +2,7 @@ import expect from 'expect'
 import React from 'react'
 import Router from '../Router'
 import ReactDOM from 'react-dom'
+import { createMemoryHistory } from 'history'
```

Comment:
Let's `import createHistory from 'history/createMemoryHistory'` to be consistent with how we're doing this elsewhere.